packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -219,33 +226,133 @@ class _ProductPageState extends State<NewProductPage> {
-          Align(
+        insets: EdgeInsets.zero, // Zero padding for the card content.
+        clipBehavior: Clip.antiAliasWithSaveLayer,
+        child: Column(
+          children: <Widget>[
+            getProductMatchHeader(context),
+            Container(
+              padding: const EdgeInsets.symmetric(horizontal: 12.0),
+              child: Column(children: [
+                getProductTitleTile(context),
+                for (final Attribute attribute in scoreAttributes)
+                  ScoreAttributeCard(
+                      attribute: attribute, iconHeight: iconHeight),
+                attributesContainer,
+              ]),
+            ),
+          ],
+        ),
+      ),
+    );
+    return ListView(children: listItems);
+  }
+
+  Widget getProductMatchHeader(BuildContext context) {
+    // NOTE: This is temporary and will be updated once the feature is supported
+    // by the server.
+    return Container(
+      color: Colors.red,
+      padding: EdgeInsets.zero,
+      child: Container(
+        alignment: Alignment.topLeft,
+        padding: const EdgeInsets.symmetric(vertical: 8),
+        child: Center(child: Text('Very poor Match', style: Theme.of(context).textTheme.subtitle1)),
+      ),
+    );
+  }
+
+  Widget getProductTitleTile(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context)!;
+    final ThemeData themeData = Theme.of(context);
+    return Align(
+      alignment: Alignment.topLeft,
+      child: ListTile(
+        contentPadding: EdgeInsets.zero,
+        title: Text(
+          _getProductName(appLocalizations),
+          style: themeData.textTheme.headline4,
+        ),
+        subtitle: Text(_product.brands ?? appLocalizations.unknownBrand),
+        trailing: Text(
+          _product.quantity ?? '',
+          style: themeData.textTheme.headline3,
+        ),
+      ),
+    );
+  }
+
+  Widget getAttributeGroupWidget(
+      BuildContext context, Product product, String groupId) {
+    final Iterable<AttributeGroup> groupIterable = _product.attributeGroups!
+        .where((AttributeGroup group) => group.id == groupId);
+    if (groupIterable.isEmpty) {
+      return emptyWidget_;
+    }
+    final AttributeGroup group = groupIterable.single;
+    return Container(
+      alignment: Alignment.topLeft,
+      child: Column(
+        children: <Widget>[
+          Container(
             alignment: Alignment.topLeft,
-            child: ListTile(
-              contentPadding: EdgeInsets.zero,
-              title: Text(
-                _getProductName(appLocalizations),
-                style: themeData.textTheme.headline4,
-              ),
-              subtitle: Text(_product.brands ?? appLocalizations.unknownBrand),
-              trailing: Text(
-                _product.quantity ?? '',
-                style: themeData.textTheme.headline3,
-              ),
+            padding: const EdgeInsets.symmetric(vertical: 8),
```

Comment:
nit: `8.0`

I think this is recommended by the styleguide, but I can't seem to find a link right now.

Reply:
sounds fair